tests/distributed_grids/1d_grid.expect=build.output

```diff
@@ -0,0 +1,2 @@
+
+DEAL::Ok
```

Comment:
Does this test run now?

Reply:
it's not meant to run, it's only meant to build

Reply:
@tamiko the test output name is `1d_grid.expect=build.output`, but this test clearly runs:

```
distributed_grids/1d_grid.debug: ******    RUN failed    *******
```

Am I misunderstanding something in how files should be named?

Reply:
What I was intending to suggest is
```
 int main(int argc, char *argv[])
 {
   deal_II_exceptions::disable_abort_on_exception();
   Utilities::MPI::MPI_InitFinalize mpi_initialization (argc, argv, 1); 
   initlog();

   try
     {
       parallel::distributed::Triangulation<1> tr(MPI_COMM_WORLD);
     }
   catch (const std::exception &exc)
     {
       deallog << exc.what() << std::endl;
     }
 }
```
and run the test in `Debug` mode, i.e. rename the output file to `1d_grid.debug.output`.